app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
+  belongs_to :post, class_name: 'Post', foreign_key: 'post_id'
+
+  def update_comments_counter
+    post.increment!(:comments_counter)
+  end
+end
```

Comment:
- [ ] The same here, Great job creating a method that updates the comments counter for a post. 👏🏻 👏🏻  However It would be nice to use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) -* `after_save` to call the `#update_comments_counter` method every time a new like has been saved to the database, then make the `comments_posts_counter` method to be a `private` method 👍🏻 💯 
- 🤔  Your code should look like this 👇🏻 :

```ruby
# app/models/comment.rb
...
     after_save : update_comments_counter
   
     private

    def update_comments_counter
       post.increment!(:likes_counter)
    end
```

Reply:
### All requests should be marked as optional except the first, but the thing is, on my dashboard, it's the first code review request, that's why I consider this PR as the first request. Hope this makes sense